src/mlpack/methods/ann/ffn.hpp

```diff
@@ -88,6 +88,23 @@ class FFN
       OutputLayerType&& outputLayer = OutputLayerType(),
       InitializationRuleType initializeRule = InitializationRuleType());
 
+  /**
+   * Create the FFN object with the given predictors and responses set (this is
+   * the set that is used to train the network) and the given optimizer.
+   * Optionally, specify which initialize rule and performance function should
+   * be used.
```

Comment:
It seems like there is no optimization happening with this constructor, so I think the comment is inaccurate.  (I guess this applies to the previous constructor too).